src/components/table/OcTableFiles.vue

```diff
@@ -711,64 +711,210 @@ export default {
       selected: []
     }),
     computed: {
-      resources() {
-        return [
-          {
-            name: "forest.jpg",
-            path: "images/nature/forest.jpg",
-            thumbnail: "https://cdn.pixabay.com/photo/2015/09/09/16/05/forest-931706_960_720.jpg",
-            indicators: [],
-            type: "file",
-            sdate: "Mon, 11 Jan 2021 14:34:04 GMT",
-            owner: [{
+          groupingSettings(){
+      return {
+        groupingAllowed: true,
+        defaultGroupingBy: "None",
+        passedGroupingBy: "",  //None, creation, owner,alphabetically
+        showGroupingOptions: true,
```

Comment:
What does this option do? Can't we just check if we have grouping functions to enable/disable the functionality?

Reply:
showGroupingOptions is needed to enable/disable oc-select element with the listed grouping options inside the component. You may want to create your own selection element from outside of the table component (what we discussed), in that case you don't pass the prop or pass "false"

Reply:
Understood!